README.md

```diff
@@ -1,121 +1,70 @@
 # CDF Fabric replicator
```

Comment:
Do we need a section explaining how the automatic time series subscription works? That is a bit different from other extractors

Reply:
That is already covered in, "Setting up Data Point Subscriptions"